HelpSource/Reference/ServerPluginAPI.schelp

```diff
@@ -0,0 +1,189 @@
+title:: Server Plugin API
```

Comment:
If someone wants to do that, they are free to. Fortunately the API sufficient for plugin development is pretty small and doesn't change very often.